cgi/user.pl

```diff
@@ -433,6 +433,8 @@
 $template_data_ref->{type} = $type;
 $template_data_ref->{crm_contact_url} = get_contact_url($user_ref);
 $template_data_ref->{org_url} = ($user_ref->{org} ? "/cgi/org.pl?type=edit&orgid=$user_ref->{org}" : '');
+$template_data_ref->{localized_donate_url} = "https://world.openfoodfacts.org/donate-to-open-food-facts";
```

Comment:
The name is "localized_donate_url", but the url is not localized. (it should be localized).

We have a string with the localized urls: lang("donate_link")